cypress-io/cypress
# chore(deps): update dependency cypress-example-kitchensink to v3.0.0

### Additional details

- Updates [packages/example/package.json](https://github.com/cypress-io/cypress/blob/develop/packages/example/package.json) to [cypress-example-kitchensink@3.0.0](https://github.com/cypress-io/cypress-example-kitchensink/releases/tag/v3.0.0).

There are no changes in the examples compared to the previously configured [cypress-example-kitchensink@2.0.7](https://github.com/cypress-io/cypress-example-kitchensink/releases/tag/v2.0.7).

### Steps to test

Ubuntu `22.04.4` LTS, Node.js `18.17.1`

Prepare unconfigured project using Node.js `20.14.0` LTS:

```bash
mkdir cy-test
cd cy-test
git init
npm init -y
npm install cypress
```

Switch to Node.js `18.17.1`

Build and run Cypress
`cd` to cypress clone

```shell
yarn
yarn start
```

In Cypress Launchpad select project `cy-test` from above.
Select E2E Testing
Select Continue
Select Electron
Select Start E2E Testing in Electron
Select "Scaffold example specs"
Select "Okay, I got it!"
Close Cypress Runner and Cypress Dashboard

`cd` to `cy-test` project and switch to Node.js `20.14.0` LTS

```shell
npx cypress run
```

All tests should run successfully.

### How has the user experience changed?

No change to user experience. Housekeeping update to use latest version of [cypress-example-kitchensink](https://github.com/cypress-io/cypress-example-kitchensink) only.

### PR Tasks

[na] Have tests been added/updated?
[na] Has a PR for user-facing changes been opened in cypress-documentation?
[na] Have API changes been updated in the type definitions?
